lib/cisco_node_utils/feature.rb

```diff
@@ -23,6 +23,25 @@ class Feature < NodeUtil
     # however, for test purposes it is sometimes convenient to support
     # feature disablement for cleanup purposes.
     # ---------------------------
+    def self.bfd_enable
+      return if bfd_enabled?
+      config_set('feature', 'bfd', state: '')
+    end
+
+    def self.bfd_disable
+      return unless bfd_enabled?
+      config_set('feature', 'bfd', state: 'no')
```

Comment:
As I noted in my email: why do we need this method at all? I'd prefer to remove it to discourage its use.